requirements.txt

```diff
@@ -11,7 +11,10 @@ Flask-Cors==3.0.8
 future==0.17.1
 gitdb2==2.0.5
 GitPython==2.1.11
+<<<<<<< HEAD
```

Comment:
Please remove the merge leftover.

Reply:
Done. Won't be there in the next update.